spec/car_spec.rb

```diff
@@ -3,38 +3,44 @@
 
 RSpec.describe Car do
   it 'exists' do
-    car = Car.new("Ford Mustang", 1500, 36)
+    car = Car.new("Ford", "Mustang", 1500, 36)
 
     expect(car).to be_a Car
   end
 
   it 'has a make' do
-    car = Car.new("Ford Mustang", 1500, 36)
+    car = Car.new("Ford", "Mustang", 1500, 36)
 
     expect(car.make).to eq("Ford")
   end
 
   it 'has a model' do
-    car = Car.new("Ford Mustang", 1500, 36)
+    car = Car.new("Ford", "Mustang", 1500, 36)
 
     expect(car.model).to eq("Mustang")
   end
 
   it 'has a monthly payment' do
-    car = Car.new("Ford Mustang", 1500, 36)
+    car = Car.new("Ford", "Mustang", 1500, 36)
 
     expect(car.monthly_payment).to eq(1500)
   end
 
   it 'has a loan length' do
-    car = Car.new("Ford Mustang", 1500, 36)
+    car = Car.new("Ford", "Mustang", 1500, 36)
 
     expect(car.loan_length).to eq(36)
   end
 
   it 'has a total cost' do
-    car = Car.new("Ford Mustang", 1500, 36)
+    car = Car.new("Ford", "Mustang", 1500, 36)
 
-    expect(car.total_cost).to eq(54000)
+    expect(car.total_cost(54000)).to eq(54000)
```

Comment:
`#total_cost` should not take an argument, it should only return a result. The method should perform a calculation based on monthly cost and months (cost * months). Looks like your first approach was closer here.

Reply:
This is what the interaction pattern said: 
```
pry(main)> car.total_cost
#=> 54000
```

Notice that `car.total_cost` does not have `()` after it - so it won't take an argument. It only returns a result.